### ISSUE TITLE
INTEF style. Accessibility issues and position problem.

### ISSUE THREAD

**Issue by ignaciogros** [exelearning/iteexe#667]
**Problems:**

- AA: Check the color/background contrast.
- Menu link over the page title (long titles only, when there's no search bar).
- ¿Bigger line-height and letter-spacing for the "Easier to read" blocks in the UDL iDevice?
- BOBY background position: _intef_footer.png should be at the page bottom.

**Solutions:**

content.css

```
/* UDL iDevice */
.exe-udlContent-content-simplified{line-height:2.5em;letter-spacing:.15em;word-spacing:.15em}
.exe-udlContent-content-simplified p{margin:2em 0}
/* WCAG */
h3{color:#576F72}
.exe-figure, .exe-figure a{color:#6F6F6F}
.exe-tabs a{color:#005D78}
#packageLicense a{color:#FFEA92}
```

nav.css

```
/* WCAG */
#header-options a{color:#547276} /* _intef_menu.gif too */
.exe-client-search-result{background:#FFEA92}
/* Fix eXe 2.8 */
#nodeTitle{padding-right:85px}
.exe-search-bar #nodeTitle{padding-right:0}
@media all and (max-width: 768px) {
	#nodeTitle{padding-right:0}
}
body{background:url(_intef_header.png) 50% 70px no-repeat,url(_intef_footer.png) center bottom no-repeat}
```
